src/xdocs/config_javadoc.xml

```diff
@@ -814,6 +736,47 @@ public int checkReturnTag(final int aTagIndex,
   &lt;property name="excludeScope" value="protected"/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          To configure the check to validate <code>throws</code> tags, you can use following config.
+          ATTENTION: checkstyle do not have information about hierarchy of exception types so usage
+          of base classes will be considered as separate exception type. As workaround you need to
+          specify both types base and exact type in javadoc.
+        </p>
+        <source>
+&lt;module name="JavadocMethod"&gt;
+  &lt;property name="validateThrows" value="true"/&gt;
+&lt;/module&gt;
+        </source>
+        <source>
+/**
+ * Actually throwed is child class of class that is declared in throws.
+ * It is limitation of checkstyle (as we do not know type hierarchy).
+ * Code and javadoc is valid BUT checkstyle can not distinguish it.
+ * @param file some file
+ * @throws IOException if some problem
+ */
+public void doSomething8(File file) throws IOException {
+    if (file == null) {
+        throw new FileNotFoundException(); // violation
+    }
+}
+
+/**
+ * Special throw type referencing in javadoc even first is parent of second type
+ * It is limitation of checkstyle (as we do not know type hierarchy).
+ * This javadoc is valid for checkstyle.
```

Comment:
Is it valid only for Checkstyle or will javadoc tool be ok with it too? If tool is ok with it, we should specify that too.

Reply:
yes it is ok by tool.
```
$ javadoc src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodExtraThrows.java
Loading source file src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodExtraThrows.java...
Constructing Javadoc information...
Standard Doclet version 1.8.0_162
Building tree for all the packages and classes...
Generating ./com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodExtraThrows.html...
Generating ./com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/package-frame.html...
Generating ./com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/package-summary.html...
Generating ./com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/package-tree.html...
Generating ./constant-values.html...
Building index for all the packages and classes...
Generating ./overview-tree.html...
Generating ./index-all.html...
Generating ./deprecated-list.html...
Building index for all classes...
Generating ./allclasses-frame.html...
Generating ./allclasses-noframe.html...
Generating ./index.html...
Generating ./help-doc.html...
```
output:
![image](https://user-images.githubusercontent.com/812984/70957034-a6a80a80-2029-11ea-879a-f8c1ea295e0b.png)

Reply:
done.